package.json

```diff
@@ -91,14 +91,15 @@
     "body-parser": "^1.16.1",
     "bootstrap": "^3.3.7",
     "boron": "^0.2.3",
+    "config": "^1.25.1",
```

Comment:
Agreed -- I missed that. For some reason I saw the `import x from y` format and I completely forgot you could import packages that way too. (I thought you were importing the json file directly.)